JSIL.Libraries/Includes/Core/Reflection/Classes/System.Type.js

```diff
@@ -180,7 +183,11 @@
       new JSIL.MethodSignature(memberArray, [$jsilcore.TypeRef("System.Reflection.BindingFlags")]),
       function (flags) {
         return JSIL.GetMembersInternal(
-          this, flags
+          this,
+          flags,
+          undefined,
```

Comment:
I generally don't like seeing undefined literals, because they're indistinguishable from an uninitialized variable being passed to a function call. Is undefined the only correct thing in these cases?